### DISCUSSION TITLE
Very beginner user. Problems running container in Docker

### DISCUSSION THREAD

**Discussion by agusbuffa** [mdolab/MACH-Aero#138] [Q&A] [answered]
Hi, I'm an aerospace engineering student from Argentina and I want to get in touch with this software. I'm researching about adaptive airfoils so optimization is a must regarding the topic. I am new with Ubuntu, Docker and all that, I find the installation tutorial a bit confusing. When I execute the command <docker run -it --name <NAME> --mount "type=bind,src=<HOST_DIR>,target=<MOUNT_DIR>" <IMAGE> /bin/bash> Docker is unable to run the container "image error response from daemon...", I replaced both DIR, the image TAG and gave a name to the container, before all this, I logged in with my Docker account.

Thank you for your time.

**Comment by AbhijithMoni** [comment, marked as answer]
Use the following command to run your Docker image. Replace `'mdolab'` with your desired name, `'E:\MDO\CRM_Wing'` with the source path on your host machine, and `'u18-gcc-ompi-latest'` with your image ID.

`$ docker run -it --name mdolab --mount "type=bind,src=E:\MDO\CRM_Wing,target=/home/mdolabuser/mount/" mdolab/public:u18-gcc-ompi-latest /bin/bash`

If you have a docker desktop running,  you can monitor the creation of a new container named `'mdolab'`.

Once the Docker container is running, navigate to the mounted directory.

`$cd mount`

List the files inside your mounted directory.

`$ls`


**Image ID**

![image](https://github.com/mdolab/MACH-Aero/assets/94369137/a2111abe-e2c9-441a-9203-59ab6c84ce0e)


Hope this helps.

Abhijith

**Reply by agusbuffa** [reply]
Thanks a lot! I could run a container (I guess). I wasn't wirting the tag well. Now, I have to confront the software.
